integration/fixtures/https/dynamic_https_sni.toml

```diff
@@ -0,0 +1,16 @@
+logLevel = "DEBUG"
```

Comment:
Why checkNewVersion is not set to false here ? https://github.com/containous/traefik/pull/2233/files#diff-89de2ef9521f40748c8902c1268dbd92R2

Reply:
`checkNewVersion = false` added :wink: